docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -55,6 +55,11 @@ const INTERFACES_WITH_DEDICATED_PAGES = [
   'GridColDef',
   'GridCsvExportOptions',
   'GridPrintExportOptions',
+
+  // Filters
+  'GridFilterModel',
+  'GridFilterItem',
+  'GridFilterOperator',
```

Comment:
The following question is just for me to learn.
You added these 3 lines and it is the doc generation script that generated the other files. Or did you modify some other files by hand to add the 3 new pages?

Reply:
Which files are you talking about for instance ?

Reply:
- docs/src/pages.js
- docs/pages/api-docs/data-grid/grid-filter-operator.js 
- docs/pages/api-docs/data-grid/grid-filter-operator.md
- docs/pages/api-docs/data-grid/grid-filter-api.json
- docs/pages/api-docs/data-grid/grid-api.md

Reply:
`pages.js` is manually authored (in the core too I believe, which is not ideal but it would probably be a pain to automate)
The rest is autogenerated by `buildInterfaceDocumentation` based on the list you commented :+1: